Mutagen.Bethesda.Analyzers.Skyrim/Record/Dialog/Topic/TooLongAnalyzer.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Mutagen.Bethesda.Analyzers.SDK.Analyzers;
+using Mutagen.Bethesda.Analyzers.SDK.Results;
+using Mutagen.Bethesda.Analyzers.SDK.Topics;
+using Mutagen.Bethesda.Skyrim;
+using Noggog;
+namespace Mutagen.Bethesda.Analyzers.Skyrim.Record.Dialog.Topic;
+
+public class TooLongAnalyzer : IIsolatedRecordAnalyzer<IDialogTopicGetter>
+{
+    private const int DialogPromptLengthLimit = 80;
+    public static readonly TopicDefinition<string, int> TopicPromptTooLong = MutagenTopicBuilder.DevelopmentTopic(
+            "Topic Prompt Too Long",
+            Severity.Suggestion)
+        .WithFormatting<string, int>("Topic prompt '{0}' is {1} longer than the recommended limit " + DialogPromptLengthLimit);
+
```

Comment:
It's a limit set in the CK by default. But it can be overridden via ini tweak. Technically more is possible but it is a recommendation by Bethesda so it fits on the screen, is consistent and so on.